cmd/opampsupervisor/supervisor/config/config.go

```diff
@@ -5,8 +5,26 @@ package config
 
 // Supervisor is the Supervisor config file format.
 type Supervisor struct {
-	Server *OpAMPServer
-	Agent  *Agent
+	Server       *OpAMPServer
+	Agent        *Agent
+	Capabilities *Capabilities `mapstructure:"capabilities"`
+}
+
+// Capabilities is the set of capabilities that the Supervisor supports.
+type Capabilities struct {
```

Comment:
Can we limit this for now to only the capabilities that are actually implemented? For example we don't yet implement `AcceptsPackages`.

I think it would be nice to start with what already works and expand this list as we implement more capabilities.